main.tf

```diff
@@ -85,6 +101,26 @@ resource "confluent_api_key" "admin_api_key" {
   ]
 }
 
+# Users Role Bindings
+data "confluent_user" "rbac_users" {
```

Comment:
I'm assuming this resource only pulls users that have already been invited to the org, but doesn't hurt to double check, is that the case?

Reply:
You assumption is correct. If user doesn't exist plan should fail.

Reply:
Tested it. If user doesn't exist Terraform throws an error like:
```
Error: error reading User: User with "email"="user@example.tld" was not found
```
which is correct, I think.